### ISSUE TITLE
[Batch] Commandline argument of New-AzBatchTask should accept empty

### ISSUE THREAD

**Issue by bgklein** [Azure/azure-powershell#9938]
Adding a new task should allow empty commandline
```
PS C:\Users\brklein> New-AzBatchTask -JobId testcase -Id task -CommandLine ""
New-AzBatchTask : Cannot validate argument on parameter 'CommandLine'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
At line:1 char:55
+ New-AzBatchTask -JobId testcase -Id task -CommandLine ""
+                                                       ~~
    + CategoryInfo          : InvalidData: (:) [New-AzBatchTask], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Microsoft.Azure.Commands.Batch.NewBatchTaskCommand
 ```

**Comment by bgklein** [comment]
@matthchr are you able to fix this in your next powershell release

**Comment by matthchr** [comment]
Yes

**Comment by matthchr** [comment]
This should be fixed by #10001
